repose-aggregator/components/filters/rate-limiting-filter/src/main/java/org/openrepose/filters/ratelimiting/write/LimitsResponseMimeTypeWriter.java

```diff
@@ -20,28 +20,44 @@
 package org.openrepose.filters.ratelimiting.write;
 
 import org.openrepose.filters.ratelimiting.util.LimitsEntityStreamTransformer;
+import org.springframework.http.MediaType;
 
-import javax.ws.rs.core.MediaType;
 import java.io.ByteArrayInputStream;
 import java.io.IOException;
 import java.io.OutputStream;
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
 
 public class LimitsResponseMimeTypeWriter {
 
+    public static final Set<MediaType> SUPPORTED_MEDIA_TYPES = Stream.of(
+            MediaType.APPLICATION_JSON,
+            MediaType.APPLICATION_XML
+            // TODO: Add support for MediaType.TEXT_XML
+    ).collect(Collectors.toSet());
```

Comment:
I moved the supported media types to this class, since it does the actual writing to the body.